buildSrc/src/main/java/Dependencies.kt

```diff
@@ -4,9 +4,8 @@
 
 object Versions {
     const val kotlin = "1.3.30"
-    const val coroutines = "1.3.1"
+    const val coroutines = "1.3.3"
```

Comment:
No action needed. To state my understanding, in [the coroutines CHANGELOG](https://github.com/Kotlin/kotlinx.coroutines/blob/master/CHANGES.md#version-133), this fix is:

> Avoid ServiceLoader for loading Dispatchers.Main (#1572, #1557, #878, #1606).

sgtm.